packages/dialtone-vue2/vite.config.js

```diff
@@ -10,21 +10,53 @@ const __dirname = path.dirname(__filename);
 export default defineConfig({
   build: {
     lib: {
-      entry: {
-        'dialtone-vue': resolve(__dirname, './index.js'),
-        emoji: resolve(__dirname, './emoji.js'),
-        message_input: resolve(__dirname, './message_input.js'),
-        directives: resolve(__dirname, './directives.js'),
-      },
+      entry: resolve(__dirname, './index.js'),
+      name: 'DialtoneVue',
+      fileName: 'dialtone-vue',
+      formats: ['es', 'cjs'],
     },
     rollupOptions: {
       external: [
+        /@dialpad/,
+        /@linusborg/,
+        /@tiptap/,
+        /date-fns/,
+        /emoji-regex/,
+        /emoji-toolkit/,
+        /tippy\.js/,
+        /prosemirror/,
         'vue',
```

Comment:
If we externalize all these dependencies doesn't that mean the consumer will have to manually install these in order to run the library?

Reply:
Those are already listed under `dependencies` so npm/pnpm/yarn will install `dependencies` and `peerDependencies`

Reply:
Could also use a vite plugin so we don't have to update this manually: https://www.npmjs.com/package/vite-plugin-externalize-deps

Reply:
Tried that plugin and didn't worked well, maybe I was doing something wrong, I'll give it another shot.

Also tried importing dependencies and peerDependencies from package.json but needed to use `assert { type: 'json' }` which is too experimental to rely on it.

Reply:
Yeah it's fine to do manually for now. not that big of deal